lucille-core/src/main/java/com/kmwllc/lucille/core/Indexer.java

```diff
@@ -183,7 +183,7 @@ private void checkForDoc() {
       sendToIndexWithAccounting(batch.flushIfExpired());
       return;
     }
-
+    log.debug("Indexing Doc {}", doc.getId());
```

Comment:
the call to sendToIndexWithAccounting is not necessarily the time when the doc is sent to the search engine
batch.add(doc) will typically add the doc to a batch being built up, and return an empty list so the sendToIndexWithAccounting call is effectively a no-op
batch.add(doc) returns a non-empty list only in certain situations, like when the batch is full and ready to be flushed